proposals/0456-remove-where-for-set-and-dictionary.md

```diff
@@ -0,0 +1,148 @@
+# Add `remove(where:)` Method for `Set` and `Dictionary`
+
+* **Proposal**: [SE-0456](0456-remove-where-for-set-and-dictionary.md)
```

Comment:
proposal numbers are assigned by the review manager when review starts. The text (and filename) should be 0000- or NNNN- for now to avoid confusion.

Reply:
pushed it.